pandas/tests/internals/test_internals.py

```diff
@@ -1310,3 +1311,14 @@ def test_make_block_no_pandas_array():
     result = make_block(arr.to_numpy(), slice(len(arr)), dtype=arr.dtype)
     assert result.is_integer is True
     assert result.is_extension is False
+
+
```

Comment:
I don't think you actually need this test; rather  in the test right above, check the result.values is is the correct type

Reply:
this test was already present, was this valid only prior to the decision of converting PandasArray to numpy array?

Reply:
I don't think that wouuld quite hit the issue. I think you would need to add a new `result = make_black(arr, ..., ndim=2)` with the right placement. It wouldn't hurt to add that, but I think keep the test below.